### ISSUE TITLE
New method: AOS.destroy()

### ISSUE THREAD

**Issue by eliamartani** [michalsnik/aos#360]
## This is:
- Feature request

## Detailed Description
Other plugins might clean `document.body` for some reason and keep `window` and `document` event listeners open.

To prevent any issues concerning it, i'm opening this request to a new feature containing a `destroy` method inside AOS, to clean it's own listeners.

The developer will be able to clean AOS listeners before any change at DOM.

## Possible Solution
```javascript
AOS.destroy();
```

**Comment by tobiasger** [comment]
Any update on when this will be available? :)

**Comment by tomgreenhill** [comment]
Any news on this being added? I seem to have run in to a problem that would benefit from this API method being available. Thanks!

**Comment by eliamartani** [comment]
> Any news on this being added? I seem to have run in to a problem that would benefit from this API method being available. Thanks!

Hi @maketimetodesign I think is better if you just fork it and implement it on your own.

**Comment by tomgreenhill** [comment]
Fair enough nice one @eliamartani will give it a shot, thanks!

**Comment by eliamartani** [comment]
@maketimetodesign I tried opening a PR but closed since there was no interaction since then.

https://github.com/michalsnik/aos/pull/361

Since the last commit in the `master` was in 2018, don't think there will be any update again.

**Comment by tomgreenhill** [comment]
Yeah think it looks that way eh! Shame as it seems solid. Going to take a look at what you did, thanks!
